apps/am/am-role-assignment-refresh-batch/demo-image-policy.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     hmcts.github.com/prod-automated: disabled
 spec:
   filterTags:
-    pattern: '^pr-470-[a-f0-9]+-(?P<ts>[0-9]+)'
+    pattern: '^prod-470-[a-f0-9]+-(?P<ts>[0-9]+)'
```

Comment:
Is this pattern correct, i.e. it still contains PR number?

Reply:
Have resolved error, well spotted